source/deploy/backup-disaster-recovery.rst

```diff
@@ -48,19 +46,357 @@ Automating backups for a Mattermost server provides a copy of the server's state
+
+.. tip::
+  The following architecture would be implemented when an entire region goes down. It does not cover the case when a single server/service goes down. For example:
+
+  - If a single app node goes down, follow best practices to provision a new node.
+  - If a database replica node goes down, create a new replica from AWS console. Or set a policy to do so automatically.
+
+Replicate database
+^^^^^^^^^^^^^^^^^^
+
+The next tasks include creating a global AWS Cluster.
+
+1. Select the RDS instance in the AWS Console, and expand the **Actions** menu to select **Add AWS Region**. 
+
+2. Choose the secondary region and enter the other details.
+
+.. warning::
+
+  Select the **Enable write forwarding** option on the secondary cluster to help forward write operations from secondary to primary. See the `AmazonRDS write forwarding <https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-global-database-write-forwarding-apg.html#aurora-global-database-write-forwarding-enabling-apg>`_ documentation for details.
+
+  Also verify the PostgreSQL version and ensure it allows ``write forwarding``. Not all PostgreSQL versions allow it. See the `Amazon RDS write forwarding region and version availability <https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-global-database-write-forwarding-apg.html#aurora-global-database-write-forwarding-regions-versions-apg>`_ documentation for details.
+
+You should now have a global cluster with the primary cluster in ``us-west-1``, and the secondary cluster in ``us-east-1``:
+
+.. image:: ../images/dr2.png
+   :alt: A screenshot of the AWS console with a global RDS cluster where the primary cluster is us-west-1 and the secondary cluster is us-east-1.
+
+Replicate S3 bucket
+^^^^^^^^^^^^^^^^^^^
+
+1. Create a new S3 bucket in the secondary region.
+
+2. Back in the original bucket, go to the **Properties** tab, and enable **Bucket versioning**.
+
+3. Go to the **Management** tab, scroll down to **Replication Rules**, and create a new replication rule.
+
+4. In the rule, select the source bucket, and then choose **Apply to all objects in the bucket** to replicate everything in the bucket.
+
+5. Choose the destination bucket. 
+
+6. For the IAM role, select **Create new role**.
+
+.. warning::
+
+  Select the **Replica modification sync** option for the bucket to help keep the replica and source buckets in sync with each other.
+
+7. Select **Save**. 
+
+8. Select **Yes** when prompted to start a job to replicate any existing objects to the secondary bucket or not.
+
+9. Perform these same steps on the secondary bucket.
+
+Now you have bi-directional replication working between these S3 replica and source buckets.
+
+Replicate ES/OS storage
+^^^^^^^^^^^^^^^^^^^^^^^
+
+1. To replicate ES/OS storage, set up CCR (cross-cluster replication) for AWS Opensearch with the following requirements:
+
+  - Elasticsearch 7.10 or OpenSearch 1.1 or later
```

Comment:
@agnivade - Which is intended: OpenSearch 1.1 or Opensearch 2.x? The recap content below suggests Opensearch 2.x.

Reply:
Excellent catch. The AWS [docs](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/replication.html#replication-prereqs) suggest 1.1. That's where this value came from. I think the 2.x later came from me because we have been using 2.x in our CI. Let's change to 2.x to be consistent.